theme/js/components/discussions/threads.vue

```diff
@@ -222,19 +236,21 @@ export default {
     },
     // Can be called from outside the component to start a new thread programmatically
```

Comment:
I kept the same because both have the same use case but we can move these to a stylemark documentation and update them when https://github.com/etalab/udata-front/pull/33 is merged.

Reply:
Maybe we could update the line to add the scroll into view idea:
```
Can be called from outside the component to start a new thread programmatically and scroll into view
```